src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/frequent/AddFrequentExpenseCommand.java

```diff
@@ -47,7 +52,8 @@ public CommandResult execute(Model model) throws CommandException {
         } catch (DuplicateFrequentTransactionException e) {
             throw new CommandException(e.getMessage());
         }
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        Tab tabToSwitchTo = Tab.values()[EXPENSE_TAB_INDEX.getZeroBased()];
+        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd), tabToSwitchTo);
```

Comment:
```suggestion
        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd), Tab.EXPENSES);
```
Same for `AddFrequentIncomeCommand` as well.
This will remove the need to have an `Index` field to specify which `Tab` to switch to.